src/CaptureFile/include/CaptureFile/CaptureFile.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_FILE_CAPTURE_FILE_H_
+#define CAPTURE_FILE_CAPTURE_FILE_H_
+
+#include <filesystem>
+#include <memory>
+#include <vector>
+
+#include "CaptureFile/CaptureFileSection.h"
+#include "OrbitBase/Result.h"
+
+namespace orbit_capture_file {
+
+class CaptureFile {
```

Comment:
Could you add a brief class-level comment here?

Reply:
done.